### ISSUE TITLE
problems with LOG_SETTINGS

### ISSUE THREAD

**Issue by djarecka** [sensein/voice-backend#5]
@satra - I don't know which versions you were using, but I just installed `sanic` and it didn't like your `LOG_SETTINGS`. Some of your fields were not present `sanic.log.LOGGING_CONFIG_DEFAULTS` and were causing errors. This is the version that works for me, but not sure if this is what you want, I simply removed the fields:
https://github.com/djarecka/voice-backend/blob/logger_issues/serve.py

cc: @sanuann

**Comment by satra** [comment]
search for `LOG_SETTINGS` in the sanic source tree. we should probably restrict sanic (and other dependencies) to a specific version

**Comment by djarecka** [comment]
@satra - not sure, why I should be searching. If you don't specify your `log_config`, it uses `LOGGING_CONFIG_DEFAULTS` that looks like [this](https://github.com/huge-success/sanic/blob/b63c06c75a54752d7f3115d3c635580db44b8399/sanic/log.py)

I'm not sure for which version your `log_config` works

**Comment by djarecka** [comment]
@satra - do you actually care about the fields that I removed in my branch?

**Comment by satra** [comment]
the things that you commented are exactly what i added to save the files on our mit server. probably the only thing you need to change is the path.

**Comment by djarecka** [comment]
@satra you're right, I thought for moment that it doesn't like the keys. Is the current dir fine? just `'filename': "console.log"` etc.

**Comment by djarecka** [comment]
@sanuann - [this version](https://github.com/djarecka/voice-backend/blob/logger_issues/serve.py) should work for you
